.github/workflows/package-publish.yml

```diff
@@ -17,58 +17,34 @@ jobs:
   publish:
     runs-on: [self-hosted, 1ES.Pool=release-pool, 1ES.ImageOverride=ubuntu-22.04]
     environment: ${{ inputs.channel }}
+    env:
+      PMC_CLI_BASE_URL: ${{ secrets.PMC_CLI_BASE_URL }}
```

Comment:
i would suggest using the values directly in the pipeline and just avoid passing all these args in the first place,. We gain GitOps as well!

Reply:
Good point! I'll add as many here as possible so that only the bare minimum are secrets. Thanks!